spec/document_builder_example_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe 'document_builder_example' do
+  before :all do
+    @test_manager = TestingApiOnlyfficeCom::TestManager.new(suite_name: 'Site Api', plan_name: @config.to_s)
+  end
+
+  before :each do
+    @instance = TestingApiOnlyfficeCom::TestInstance.new(@config)
+    @api_page = @instance.go_to_main_page
+  end
+
+  describe 'document_builder_generate' do
+    before :each do
+      @introduction_page = @api_page.go_to_document_builder_introduction
+    end
+
+    it 'generate document works' do
+      builder_download = @introduction_page.builder_download
+      expect(File.file?(builder_download)).to be_truthy
+      parser_builder = OoxmlParser::Parser.parse(builder_download)
+      builder_text = parser_builder.elements[2].character_style_array[0].text
+      /^ONLYOFFICE.*Commercial director.*/.match?(builder_text)
```

Comment:
`expect(builder_text).to match(/^ONLYOFFICE.*Commercial director.*/)`
Without `except` this test never fail